### PR TITLE
fix: permission check for cloning board

### PR DESCRIPTION
**Pull request by luka-nextcloud**
* Resolves: 
* Target version: main

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: src/components/navigation/AppNavigationBoard.vue [hunk 1]
```diff
@@ -253,6 +253,9 @@ export default {
 			try {
 				const newBoard = await this.$store.dispatch('cloneBoard', this.board)
 				this.loading = false
+				if (newBoard instanceof Error) {
+					throw newBoard
+				}
```
Comment (juliushaertl) [inline src/components/navigation/AppNavigationBoard.vue:258]: Can we also make sure that we hide the clone menu entry if that permission is not available?

**Review by juliushaertl — APPROVED** [review]

**Comment by luka-nextcloud** [comment]
Failed cypress tests are not related to code changes. Some other PRs have the same failed cypress tests.

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**